packages/react/src/Form/Checkbox/Client/CheckboxItem.tsx

```diff
@@ -0,0 +1,76 @@
+import { ComponentPropsWithoutRef, ReactNode, forwardRef, useId } from "react";
+import CheckBoxOutlineBlankIcon from "@mui/icons-material/CheckBoxOutlineBlank";
+import CheckBoxIcon from "@mui/icons-material/CheckBox";
+
+import "@axa-fr/design-system-css/dist/Form/Checkbox/Client/Checkbox.scss";
+
+export type CheckboxItemOption = {
+  id?: string;
+  icon?: ReactNode;
+  label: ReactNode;
+  description?: ReactNode;
+  value: string;
+  disabled?: boolean;
+};
+
+type Props = Omit<ComponentPropsWithoutRef<"input">, "type" | "label"> &
+  CheckboxItemOption & {
+    checked?: boolean;
+  };
+
+export const CheckboxItem = forwardRef<HTMLInputElement, Props>(
+  (
+    {
+      disabled = false,
+      value = "",
+      id,
+      icon,
+      label,
+      description,
+      className,
+      checked,
+      ...otherProps
+    }: Props,
+    inputRef,
+  ) => {
+    const generatedId = useId();
+    const newId = id ?? generatedId; // id is required on this component
+    const defaultClassName = "af-form-client__checkbox";
+    const getOptionClassName = () => {
+      const baseClassName = className ?? defaultClassName;
+      return disabled
+        ? `${baseClassName} ${defaultClassName}--disabled`
```

Comment:
ça ne devrait pas plutôt être ça ?
```suggestion
        ? `${baseClassName} ${baseClassName}--disabled`
```

Reply:
Bien vu. Corrigé !